### ISSUE TITLE
Error on Termux when installing plugins

### ISSUE THREAD

**Issue by jarun** [jarun/nnn#370]
While the plugins are installed, 2 errors are thrown on termux. Applies to the `suedit` and `fzcd` commands as well.

```
u0_a117@localhost ~/G/n/plugins> ./getplugs 
./getplugs: 16: [: 1: unexpected operator
./getplugs: 18: [: 1: unexpected operator
'nnn-master/plugins/.nnn-plugin-helper' -> './plugins/.nnn-plugin-helper'
'nnn-master/plugins/README.md' -> './plugins/README.md'
'nnn-master/plugins/boom' -> './plugins/boom'
'nnn-master/plugins/checksum' -> './plugins/checksum'
'nnn-master/plugins/drag-file' -> './plugins/drag-file'
'nnn-master/plugins/drop-file' -> './plugins/drop-file'
'nnn-master/plugins/dups' -> './plugins/dups'
'nnn-master/plugins/fzcd' -> './plugins/fzcd'
'nnn-master/plugins/fzy-open' -> './plugins/fzy-open'
'nnn-master/plugins/getplugs' -> './plugins/getplugs'
'nnn-master/plugins/gutenread' -> './plugins/gutenread'
'nnn-master/plugins/hexview' -> './plugins/hexview'
'nnn-master/plugins/imgresize' -> './plugins/imgresize'
'nnn-master/plugins/imgur' -> './plugins/imgur'
'nnn-master/plugins/ipinfo' -> './plugins/ipinfo'
'nnn-master/plugins/kdeconnect' -> './plugins/kdeconnect'
'nnn-master/plugins/mediainf' -> './plugins/mediainf'
'nnn-master/plugins/moclyrics' -> './plugins/moclyrics'
'nnn-master/plugins/mocplay' -> './plugins/mocplay'
'nnn-master/plugins/ndiff' -> './plugins/ndiff'
'nnn-master/plugins/nmount' -> './plugins/nmount'
'nnn-master/plugins/notes' -> './plugins/notes'
'nnn-master/plugins/nwal' -> './plugins/nwal'
'nnn-master/plugins/oldbigfile' -> './plugins/oldbigfile'
'nnn-master/plugins/organize' -> './plugins/organize'
'nnn-master/plugins/pastebin' -> './plugins/pastebin'
'nnn-master/plugins/pdfview' -> './plugins/pdfview'
'nnn-master/plugins/picker' -> './plugins/picker'
'nnn-master/plugins/pywal' -> './plugins/pywal'
'nnn-master/plugins/readit' -> './plugins/readit'
'nnn-master/plugins/ringtone' -> './plugins/ringtone'
'nnn-master/plugins/splitjoin' -> './plugins/splitjoin'
'nnn-master/plugins/suedit' -> './plugins/suedit'
'nnn-master/plugins/sxiv' -> './plugins/sxiv'
'nnn-master/plugins/thumb' -> './plugins/thumb'
'nnn-master/plugins/transfer' -> './plugins/transfer'
'nnn-master/plugins/uidgid' -> './plugins/uidgid'
'nnn-master/plugins/upgrade' -> './plugins/upgrade'
'nnn-master/plugins/vidthumb' -> './plugins/vidthumb'
'nnn-master/plugins/viuimg' -> './plugins/viuimg'
u0_a117@localhost ~/G/n/plugins>
```

**Comment by jarun** [comment]
Fixed at PR #370.

**Comment by annagrram** [comment]
> Fixed at PR #370.

You mean PR #371 ;)
